src/qibolab/instruments/zhinst.py

```diff
@@ -54,10 +54,12 @@ def select_pulse(pulse, pulse_type):
     """Pulse translation"""
 
     if str(pulse.shape) == "Rectangular()":
+        can_compress = not pulse.type is PulseType.READOUT
```

Comment:
```suggestion
        can_compress = pulse.type is not PulseType.READOUT
```
I think this is the better way to do it, sounds "more english"